client/app/utils/customTransformErrors.ts

```diff
@@ -0,0 +1,15 @@
+import { RJSFValidationError } from "@rjsf/utils";
+
+export const customTransformErrors = (
+  errors: RJSFValidationError[],
+  customFormatsErrorMessages: { [key: string]: string },
+) => {
+  return errors.map((error) => {
+    if (error.name !== "format") return error;
```

Comment:
In [transformErrors](https://github.com/bcgov/cas-registration/blob/8ad3a972150961e768945380fca9420666315163/client/app/components/form/FormBase.tsx) in my PR we rename required fields which aren't a format so this should be removed.

I don't mind using `customFormatsErrorMessages` map you've created though if you have a nice readable way to support more error types.